packages/@glimmer/test-helpers/lib/render-test.ts

```diff
@@ -992,6 +1000,19 @@ export function content(list: Content[]): string {
   return out.join('');
 }
 
+export function processAssertElementArgs(args: any[]) {
```

Comment:
Would you mind adding a few comments here describing expected inputs and outputs? It's hard for my poor brain to reverse the intended data structures from the code below.

Reply:
OK, updated with some docs (and a TODO to make it clearer that this needs future cleanup "when we have time").